lib/Parse/Lexer.cpp

```diff
@@ -1316,16 +1335,82 @@ static const char *skipToEndOfInterpolatedExpression(const char *CurPtr,
   }
 }
 
+/// getStringLiteralContent:
+/// Extract content of string literal from inside quotes.
+static StringRef getStringLiteralContent(const Token &Str) {
+  StringRef Bytes = Str.getText();
+
+  if (Str.getStringModifiers() & StringLiteralMultiline)
+    Bytes = Bytes.drop_front(3).drop_back(3);
+  else
+    Bytes = Bytes.drop_front().drop_back();
+
+  return Bytes;
+}
+
+/// getTrailingIndent:
+/// Determine contents of literal to be indent stripped.
```

Comment:
Update your comment with the new function name. Also, this description doesn't seem correct. Do you mean "Determine trailing indent to be used for multiline literal indent stripping."?